docs/includes/fundamentals/gridfs/GridFSTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use MongoDB\BSON\ObjectId;
+use MongoDB\Client as MongoDBClient;
+use MongoDB\Driver\Manager;
+use MongoDB\GridFS\Bucket;
+use MongoDB\Laravel\Tests\TestCase;
+
+use function fclose;
+use function file_get_contents;
+use function fopen;
+use function fwrite;
+use function stream_get_contents;
+
+class GridFSTest extends TestCase
```

Comment:
I don't see any reference to this file from the docs, so why is this being introduced?

The lines constructing Manager and Client instances don't use the test suites connection string, so I expect this may not run in all CI environments.